diff --git a/conf/corosync.conf.example b/conf/corosync.conf.example
index 7b3abed1..04444322 100644
--- a/conf/corosync.conf.example
+++ b/conf/corosync.conf.example
@@ -2,25 +2,52 @@
 totem {
 	version: 2
 
+	# Corosync itself works without a cluster name, but DLM needs one.
+	# The cluster name is also written into the VG metadata of newly
+	# created shared LVM volume groups, if lvmlockd uses DLM locking.
+	# It is also used for computing mcastaddr, unless overridden below.
+	cluster_name: debian
+
+	# How long before declaring a token lost (ms)
+	token: 3000
+
+	# How many token retransmits before forming a new configuration
+	token_retransmits_before_loss_const: 10
+
+	# Limit generated nodeids to 31-bits (positive signed integers)
+	clear_node_high_bit: yes
+
 	# crypto_cipher and crypto_hash: Used for mutual node authentication.
 	# If you choose to enable this, then do remember to create a shared
 	# secret with "corosync-keygen".
 	# enabling crypto_cipher, requires also enabling of crypto_hash.
+	# crypto_cipher and crypto_hash should be used instead of deprecated
+	# secauth parameter.
+
+	# Valid values for crypto_cipher are none (no encryption), aes256, aes192,
+	# aes128 and  3des. Enabling crypto_cipher, requires also enabling of
+	# crypto_hash.
 	crypto_cipher: none
+
+	# Valid values for crypto_hash are  none  (no  authentication),  md5,  sha1,
+	# sha256, sha384 and sha512.
 	crypto_hash: none
 
+	# Optionally assign a fixed node id (integer)
+	# nodeid: 1234
+
 	# interface: define at least one interface to communicate
 	# over. If you define more than one interface stanza, you must
 	# also set rrp_mode.
 	interface {
-                # Rings must be consecutively numbered, starting at 0.
+		# Rings must be consecutively numbered, starting at 0.
 		ringnumber: 0
 		# This is normally the *network* address of the
 		# interface to bind to. This ensures that you can use
 		# identical instances of this configuration file
 		# across all your cluster nodes, without having to
 		# modify this option.
-		bindnetaddr: 192.168.1.0
+		bindnetaddr: 127.0.0.1
 		# However, if you have multiple physical network
 		# interfaces configured for the same subnet, then the
 		# network address alone is not sufficient to identify
@@ -34,7 +61,7 @@ totem {
 		# the network administrator). Do not reuse multicast
 		# addresses across multiple Corosync clusters sharing
 		# the same network.
-		mcastaddr: 239.255.1.1
+		# mcastaddr: 239.255.1.1
 		# Corosync uses the port you specify here for UDP
 		# messaging, and also the immediately preceding
 		# port. Thus if you set this to 5405, Corosync sends
@@ -58,10 +85,12 @@ logging {
 	to_stderr: no
 	# Log to a log file. When set to "no", the "logfile" option
 	# must not be set.
-	to_logfile: yes
-	logfile: /var/log/cluster/corosync.log
+	to_logfile: no
+	#logfile: /var/log/corosync/corosync.log
 	# Log to the system log daemon. When in doubt, set to yes.
 	to_syslog: yes
+	# Log with syslog facility daemon.
+	syslog_facility: daemon
 	# Log debug messages (very verbose). When in doubt, leave off.
 	debug: off
 	# Log messages with time stamps. When in doubt, set to on
@@ -77,5 +106,6 @@ logging {
 quorum {
 	# Enable and configure quorum subsystem (default: off)
 	# see also corosync.conf.5 and votequorum.5
-	#provider: corosync_votequorum
+	provider: corosync_votequorum
+	expected_votes: 2
 }
