diff --git a/examples/tcpserver.c b/examples/tcpserver.c
index 88210e9..ea6140d 100644
--- a/examples/tcpserver.c
+++ b/examples/tcpserver.c
@@ -33,7 +33,7 @@
 #include <sys/socket.h>
 #endif /* HAVE_SYS_SOCKET_H */
 #ifdef HAVE_SYS_POLL_H
-#include <sys/poll.h>
+#include <poll.h>
 #endif /* HAVE_SYS_POLL_H */
 
 #include <qb/qbdefs.h>
diff --git a/include/qb/qbipcc.h b/include/qb/qbipcc.h
index 77631e0..952450b 100644
--- a/include/qb/qbipcc.h
+++ b/include/qb/qbipcc.h
@@ -30,7 +30,7 @@ extern "C" {
 #include <qb/qbconfig.h>
 
 #include <pthread.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/socket.h>
 #include <qb/qbhdb.h>
 #include <qb/qbipc_common.h>
diff --git a/lib/loop_poll_int.h b/lib/loop_poll_int.h
index f99f442..6652023 100644
--- a/lib/loop_poll_int.h
+++ b/lib/loop_poll_int.h
@@ -24,7 +24,7 @@
 #include "os_base.h"
 
 #ifdef HAVE_SYS_POLL_H
-#include <sys/poll.h>
+#include <poll.h>
 #endif /* HAVE_SYS_POLL_H */
 
 #include <qb/qbdefs.h>
diff --git a/tests/loop.c b/tests/loop.c
index 1a6d7dd..4c72b9c 100644
--- a/tests/loop.c
+++ b/tests/loop.c
@@ -19,7 +19,7 @@
  * along with libqb.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "os_base.h"
-#include <sys/poll.h>
+#include <poll.h>
 
 #include <qb/qbdefs.h>
 #include <qb/qbutil.h>
